Alignment/OfflineValidation/plugins/PixelBaryCentreAnalyzer.h

```diff
@@ -0,0 +1,117 @@
+#ifndef Alignment_OfflineValidation_PixelBaryCentreAnalyzer_H
+#define Alignment_OfflineValidation_PixelBaryCentreAnalyzer_H
+
+/** \class PixelBaryCentreAnalyzer
+ *  The analyer works as the following :
+ *  - Read global tracker position from global tag
+ *  - Read tracker alignment constants from different ESsource with different labels 
+ *  - Calculate barycentres for different pixel substructures using global tracker position and alignment constants
+ *  and store them in trees, one for each ESsource label.
+ *
+ *  Python script plotBaryCentre_VS_BeamSpot.py under script dir is used to plot barycentres from alignment constants used in Prompt-Reco, End-of-Year Rereco and so-called Run-2 (Ultra)Legacy Rereco. Options of the plotting script can be found from the helper in the script.
+ *
+ *  $Date: 2021/01/05 $ 
+ *  $Revision: 1.0 $
+ *  \author Tongguang Cheng - Beihang University <tongguang.cheng@cern.ch>
+ */
+
+// Framework
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
this should go.

Reply:
I got compile error removing Event.h header
"invalid use of incomplete type 'const class edm::Event'
  212 |   ls_ = iEvent.id().luminosityBlock()"
  and compile error removing MakerMacros header
  aggregate 'edm::ParameterSetDescription desc' has incomplete type and cannot be defined

Reply:
In case it was not clear it's just
```
#include "FWCore/Framework/interface/ESHandle.h
```
that is unnecessary after the esConsumes migration.